src/test/java/core/basesyntax/service/impl/ProcessServiceImplTest.java

```diff
@@ -0,0 +1,89 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ServiceException;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ProcessServiceImplTest {
+    private ProcessServiceImpl processService = new ProcessServiceImpl();
```

Comment:
initialize services and fields in the before-all section, check other classes as well